core/src/main/scala/org/apache/spark/scheduler/EventLoggingListener.scala

```diff
@@ -82,6 +82,10 @@ private[spark] class EventLoggingListener(
     CompressionCodec.getShortName(c.getClass.getName)
   }
 
+  private val eventLogPermission = sparkConf.get("spark.eventLog.permission", "770")
```

Comment:
Let's define the configuration around here:

https://github.com/apache/spark/blob/d0443a74d185ec72b747fa39994fa9a40ce974cf/core/src/main/scala/org/apache/spark/internal/config/package.scala#L73-L121